website/docs/r/front_door.html.markdown

```diff
@@ -0,0 +1,243 @@
+
+* `name` - (Required) The name of the `Backend Pool`.
+
+* `backend` - (Required) A `backend` block as defined below.
+
+* `load_balancing_name` - (Required) The name property of the `backend_pool_load_balancing` block whithin this resource to use for the `Backend Pool`.
+
+* `health_probe_name` - (Required) The name property of a `backend_pool_health_probe` block whithin this resource to use for the `Backend Pool`.
+
+---
+
+The `backend` block supports the following:
+
+* `address` - (Required) Location of the backend (IP address or FQDN)
+
+* `host_header` - (Required) The value to use as the host header sent to the backend.
+
+* `http_port` - (Required) The HTTP TCP port number. Possible values are between `1` - `65535`.
+
+* `https_port` - (Required) The HTTPS TCP port number. Possible values are between `1` - `65535`.
+
+* `priority` - (Optional) Priority to use for load balancing. Higher priorities will not be used for load balancing if any lower priority backend is healthy. Defaults to `1`.
+
+* `weight` - (Optional) Weight of this endpoint for load balancing purposes. Defaults to `50`.
+
+---
+
+The `frontend_endpoint` block supports the following:
+
+* `name` - (Required) The name of the Frontend Endpoint.
+
+* `host_name` - (Required) The host name of the Frontend Endpoint. Must be a domain name.
+
+* `session_affinity_enabled` - (Optional) Whether to allow session affinity on this host. Valid options are `true` or `false` Defaults to `false`.
+
+* `session_affinity_ttl_seconds` - (Optional) The TTL to use in seconds for session affinity, if applicable. Defaults to `0`.
+
+* `enable_custom_https_provisioning` - (Required) Name of the Frontend Endpoint.
+
+---
+
+The `backend_pool_health_probe` block supports the following:
+
+* `name` - (Required) The name of the Azure Front Door Backend Health Probe.
+
+* `path` - (Optional) The path to use for the Backend Health Probe. Default is `/`.
+
+* `protocol` - (Optional) Protocol scheme to use for the Backend Health Probe. Defaults to `Http`.
+
+* `interval_in_seconds` - (Optional) The number of seconds between health probes. Defaults to `120`.
+
+---
+
+The `backend_pool_load_balancing` block supports the following:
+
+* `name` - (Required) The name of the Azure Front Door Backend Load Balancer.
+
+* `sample_size` - (Optional) The number of samples to consider for load balancing decisions. Defaults to `4`.
+
+* `successful_samples_required` - (Optional) The number of samples within the sample period that must succeed. Defaults to `2`.
+
+* `additional_latency_milliseconds` - (Optional) The additional latency in milliseconds for probes to fall into the lowest latency bucket. Defaults to `0`.
+
+---
+
+The `routing_rule` block supports the following:
+
+* `name` - (Required) The name of the Front Door Backend Routing Rule.
+
+* `frontend_endpoints` - (Required) The name property of the `frontend_endpoint` block whithin this resource to associate with this `routing_rule`.
```

Comment:
Could we elaborate that multiple can go here? I think this might be worded better as
```suggestion
* `frontend_endpoints` - (Required) The names of the `frontend_endpoint` blocks whithin this resource to associate with this `routing_rule`.
```

or

```suggestion
* `frontend_endpoints` - (Required) The name properties of the `frontend_endpoint` blocks whithin this resource to associate with this `routing_rule`.
```